### PR TITLE
Enable the ordering of logs

### PR DESCRIPTION
**Pull request by rjzondervan**

### CODE REVIEW

**Review by WilcoLouwerse — APPROVED** [review]
This feels like partly duplicate code with the CoreBundle CacheService way of finding MongoDB objects.
Maybe we should consider reusing CoreBundle code in this controller

**Comment by rjzondervan** [comment]
> This feels like partly duplicate code with the CoreBundle CacheService way of finding MongoDB objects. Maybe we should consider reusing CoreBundle code in this controller

This is very much duplicate code indeed. We should indeed look for a way to merge the code for logs and objects. Because for the moment this is code split between gateway and corebundle this is the only way to get it working for now.

File: api/src/Controller/LogController.php [hunk 1]
```diff
@@ -130,7 +130,15 @@ public function logAction(Request $request, RequestService $requestService)
             $filter['_extend'], $filter['_search'], $filter['_order'], $filter['_fields']);
         $completeFilter = $this->setPagination($limit, $start, $completeFilter);
 
-        $content = json_encode(['results' => $collection->find($filter, ['limit' => $limit, 'skip' => $start])->toArray(), 'page' => intval($completeFilter['_page'] ?? 1), 'count' => $total = $collection->count($filter), 'pages' => floor($total / $limit)]);
+        $order = isset($completeFilter['_order']) === true ? str_replace(['ASC', 'asc', 'DESC', 'desc'], [1, 1, -1, -1], $completeFilter['_order']) : [];
+        !empty($order) && $order[array_keys($order)[0]] = (int) $order[array_keys($order)[0]];
```
Comment (bbrands02) [inline api/src/Controller/LogController.php:134]: empty($order) === false ipv !empty for codacy?

**Review by bbrands02 — APPROVED** [review]
Approved knowing this will be later refactored

**Comment by rjzondervan** [comment]
I merge this pull request knowing it needs some refactoring on the short term. Because we want to merge the mongodb functionalities to one more abstract service we chose not to refactor this immediately, but when the both mongodb-dependent functionalities are merged into one service.

**Pull request merged**